### ISSUE TITLE
proc missing a return statement in body part code?

### ISSUE THREAD

**Issue by ATH1909** [tgstation/tgstation#46064]
I was codediving in body part code and found this:
```
/obj/item/bodypart/proc/is_disabled()
	if(HAS_TRAIT(src, TRAIT_PARALYSIS))
		return BODYPART_DISABLED_PARALYSIS
	if(can_dismember() && !HAS_TRAIT(owner, TRAIT_NOLIMBDISABLE))
		. = disabled //inertia, to avoid limbs healing 0.1 damage and being re-enabled
		if((get_damage(TRUE) >= max_damage) || (HAS_TRAIT(owner, TRAIT_EASYLIMBDISABLE) && (get_damage(TRUE) >= (max_damage * 0.6)))) //Easy limb disable disables the limb at 40% health instead of 0%
			return BODYPART_DISABLED_DAMAGE
		if(disabled && (get_damage(TRUE) <= (max_damage * 0.5)))
			return BODYPART_NOT_DISABLED
	else
		return BODYPART_NOT_DISABLED
```
Correct me if I'm wrong here, but if the limb doesn't have the TRAIT_PARALYSIS trait, can_dismember() returns true, the owner does NOT have the TRAIT_NOLIMBDISABLE trait, the damage of the limb is less than max_damage, the owner does NOT have the TRAIT_EASYLIMBDISABLE trait (or if they do but the damage of the limb is less than max_damage*0.6), and either the limb was either not already disabled (in other words, the "disabled" variable is false) or the limb's damage is less than max_damage*0.5, this proc should end without hitting a return statement, correct?

In other words, if Sally J. Test has no damage on her arm and she doesn't have any weird stuff applying to her or her arm, is_disabled() shouldn't hit any return statements by the time that it ends, correct? Isn't that a bad thing?

**Comment by ATH1909** [comment]
Nevermind, I've been informed that this isn't actually a problem (. is set to disabled).
